apps/test/integration/util/runLevelTest.js

```diff
@@ -181,6 +181,9 @@ function runLevel(app, skinId, level, onAttempt, testData) {
     },
     onAttempt: onAttempt
   });
+  if (app === 'gamelab') {
+    window.Gamelab = loadedApp;
```

Comment:
Instead of doing a gamelab special-case here in `runLevelTest`, you could do this in `loadTestableGamelab.js`.  Thoughts?